packages/grafana-ui/src/components/Divider/Divider.tsx

```diff
@@ -0,0 +1,38 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+
+import { useTheme2 } from '../../themes';
+
+interface Props {
```

Comment:
If you'd like to remove the breaking change status on the PR you can rename this interface to something other than props. Maybe`DividerProps`?

Sorry for the bother - there's a bug in our tooling (Levitate) that looks for breaking changes in PRs and duplicate types / interfaces with the same name (in this case `Props`) create false positives.

Reply:
Got it, thanks for that, I made the changes now.

Reply:
Awesome stuff! 🚀